lib/tasks/sync_governance.rake

```diff
@@ -2,7 +2,7 @@ require "task_lock"
 
 task sync_governance: :environment do
   $stdout.sync = true
-  TaskLock.with_lock!(:sync) do
+  TaskLock.with_lock!(:sync_governance) do
```

Comment:
can't do this without also updating `LOCKFILES`
https://github.com/figment-networks/bakerhub-indexer/blob/master/lib/task_lock.rb#L8